packages/mask/src/plugins/Tips/components/TipButton.tsx

```diff
@@ -81,11 +83,15 @@ export const TipButton: FC<Props> = ({
         return NextIDProof.queryIsBound(receiverPersona.publicHexKey, platform, receiver.userId, true)
     }, [receiverPersona?.publicHexKey, platform, receiver?.userId])
 
-    const [walletsState, queryBindings] = useAsyncFn(async () => {
+    const [publicKey, setPublicKey] = useState<string | null>(null)
+    const { value: kv } = useKvGet<NextIDStorageInfo<BindingProof[]>>(publicKey)
+
+    const [NextIDWalletsState, queryBindings] = useAsyncFn(async () => {
```

Comment:
```suggestion
    const [wallets, queryWallets] = useAsyncFn(async () => {
```

Reply:
I would keep `NextIDWalletsState` to identify between wallets from kv services (wallets from cloud)